src/styles/Buttons.scss

```diff
@@ -0,0 +1,34 @@
+.nav-button {
```

Comment:
It seems like some of the properties like `font-size`, `margin`, `padding`, and `text-decoration` are the same between `nav-button` and `control-button`.  Could this be abstracted into a generic class that can be shared?